express/properties/non_scalar/file_content.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, name, parser, *args, **kwargs):
         self.basename = str(kwargs['basename'])
         self.filetype = str(kwargs['filetype'])
         self.work_dir = str(kwargs['work_dir'])
```

Comment:
we can drop the self.work_dir define now, if we're using `upload_dir` instead

Reply:
work_dir attribute removed